src/features/Instructors/InstructorsFilters/index.jsx

```diff
@@ -43,7 +43,9 @@ const InstructorsFilters = ({ resetPagination }) => {
     const formJson = Object.fromEntries(formData.entries());
     dispatch(updateFilters(formJson));
     try {
-      dispatch(fetchInstructorsData(currentPage, formJson));
+      const initialPage = 1;
```

Comment:
What do you think about having a module dedicated to constants? @AuraAlba

Reply:
We already have a constants file, so I moved it there